metricbeat/module/aerospike/_meta/config.yml

```diff
@@ -3,3 +3,9 @@
   #  - namespace
   period: 10s
   hosts: ["localhost:3000"]
+
+  # Aerospike Cluster Name
+  # cluster_name: myclustername
+
+  # Read more about SSL Configurations https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-ssl.html
+  # ssl.enabled: false
```

Comment:
Sorry, I forgot about this last time.  But you need to add the new options to the `config.reference.yml` as well so it shows up in the `metricbeat.reference.yml` file.  And should add a link to the SSL options in the `docs.asciidoc` so the documentation gets updated as well.

Reply:
It makes sense :). I think I got it. I added to fields.yml the "ssl" settings and it has been generated automatically ^^. I took inspiration from this PR https://github.com/elastic/beats/commit/4db5f0aaefeb9038d7656b470a44a8cc37d19a37. I hope it looks better now!